GuessingGame.java

```diff
@@ -0,0 +1,53 @@
+#GuessingGameSolution.java
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, module expected